bentoml/_internal/server/http_app.py

```diff
@@ -264,9 +267,26 @@ def on_startup(self) -> list[t.Callable[[], None]]:
         else:
             for runner in self.bento_service.runners:
                 on_startup.append(runner.init_client)
+
         on_startup.extend(super().on_startup)
         return on_startup
 
+    async def readyz(self, _: "Request") -> "Response":
+
+        ready_status = all(
+            await asyncio.gather(
+                *(
+                    runner.runner_handle_is_ready()
+                    for runner in self.bento_service.runners
+                )
+            )
+        )
+
+        if ready_status:
+            return PlainTextResponse("\n", status_code=200)
+        else:
+            raise HTTPException(500)
```

Comment:
For this I think you can just put the ready_status logic into the if branch, and no need to have an else here, you can just raise directly after the if block:
```python
if all(...):
   return PlainTextResponse
raise ...
```

Reply:
Thanks Aaron. That does look cleaner. Just pushed.